### DISCUSSION TITLE
How to run JAR file!

### DISCUSSION THREAD

**Discussion by GoRaN9O** [REAndroid/APKEditor#10] [Q&A] [answered]
Hello,

Can anyone tell me how can I use this tool to decompile APK files!

It's only an JAR file not a standard application like windows `.EXE` or any `.MSI` known program files! I need some help please.

**Comment by REAndroid** [comment, marked as answer]
Search "How to run executable jar file"

**Reply by GoRaN9O** [reply]
Oh sorry i just tried to run like an normal windows app but the tool must be using with command and it only works in Windows PowerShell not CMD... Thanks it works perfectly :)

**Comment by GoRaN9O** [comment]
All tools are works great expect the new one (Protect) when you try to protect an APK file after build the apk file is can't install on the devices i think that because unsigned file.

**Reply by REAndroid** [reply]
> All tools are works great expect the new one (Protect) when you try to protect an APK file after build the apk file is can't install on the devices i think that because unsigned file.

Please create issue with the apk you used
